src/assets/v3/ownership-transfer.njk

```diff
@@ -0,0 +1,28 @@
+---
+layout: layouts/base.njk
+---
+<div class="o-layout__sidebar"></div>
+
+<div class="o-layout__main o-layout-typography">
+
+<h1 id="new-chapter">Polyfill.io is entering a new chapter</h1>
+
+<p>A decade ago when polyfill.io started life, polyfills were vital to the advancement of web development practice. Today, the situation is quite different, and the time lag from a browser feature being firmly defined and it being widely supported is typically small, thanks to evergreen browsers. As a result, at the FT we aren't using polyfill.io any more, and new adoption of it has declined.</p>
```

Comment:
This is not quite true, browser versions such as iOS safari still stick around and don't update immediately.

This article does a good job to explain the difference between evergreen and immediately available -- https://css-tricks.com/evergreen-does-not-mean-immediately-available/

Also, this post is about ownership transfer, not about adoption rates for the service, can we only talk about the ownership transfer please?

Reply:
Can we make this more succinct? FT no longer wants to own or maintain polyfill, ownership is being transferred to the core maintainer (Jake Champion), who has maintained the infrastructure and codebase for 7+ years